package.json

```diff
@@ -8,10 +8,12 @@
     "prettier": "3.3.3"
   },
   "scripts": {
-    "prettier:check": "prettier --check .",
+    "prettier": "prettier --check .",
     "prettier:fix": "prettier --write .",
-    "lint:check": "pnpm -r lint:check",
+    "lint": "pnpm -r lint:check",
```

Comment:
like the above, also note that you're introducing the inconsistency:
- to check linting at the root you run `lint`
- but in a package you run `lint:check`